doc/build/docker_install.rst

```diff
@@ -1,122 +1,81 @@
-Docker installation guide
-==========================
+Using and Building Docker Images
+================================
 
-PaddlePaddle provide the `Docker <https://www.docker.com/>`_ image. `Docker`_ is a lightweight container utilities. The performance of PaddlePaddle in `Docker`_ container is basically as same as run it in a normal linux. The `Docker`_ is a very convenient way to deliver the binary release for linux programs.
+We release PaddlePaddle in the form of `Docker <https://www.docker.com/>`_ images on `dockerhub.com <https://hub.docker.com/r/paddledev/paddle/>`_.   Running as Docker containers is currently the only officially-supported way to running PaddlePaddle.
 
-..  note::
+Run Docker images
+-----------------
 
-    The `Docker`_ image is the recommended way to run PaddlePaddle 
+For each version of PaddlePaddle, we release 4 variants of Docker images:
```

Comment:
The docker `demo` images should be removed, but there are many other user image will depends on `devel` images. The devel means the whole dependencies to build `Paddle` are installed.

User will use `devel` image to `rebuild` Paddle as `debug mode`, use different `cudnn`, etc. It is very useful image and please do not remove it.